migrations/sql/V2018.11.20.1106__AddModelMeta.sql

```diff
@@ -0,0 +1,60 @@
+--- What this Migration is about
+--- comment on model.is_current
+--- &
+--- creating three new table
+--- 1. model_version_meta
+--- 2. model_version_vaccine
+--- 3. model_version_country
+
+COMMENT ON COLUMN model.is_current IS
+  'TRUE if the model is currently running for VIMC.';
+
+CREATE TABLE model_version_meta (
+  id SERIAL,
+  model_version INTEGER NOT NULL,
+  is_dynamic BOOLEAN NOT NULL, 
+  code TEXT,
+  dalys BOOLEAN NOT NULL,
+  gender INTEGER NOT NULL,
+  burden_min_age INTEGER NOT NULL,
+  burden_max_age INTEGER NOT NULL,
+  cohort_first INTEGER, --- nullable, NA means 2000
+  cohort_last INTEGER, --- nullable, NA means min(2100, age_max+2030)
+  year_min INTEGER, --- nullable, NA means 2000
+  year_max INTEGER, --- nullable, NA means 2100
+  PRIMARY KEY (id),
+  FOREIGN KEY (model_version) REFERENCES model_version(id),
+  FOREIGN KEY (gender) REFERENCES gender(id)
+);
+COMMENT ON TABLE model_version_meta IS
+  'General model metadata, including whether models consider herd effect, how models are coded, whether models provide DALYs, model observation shape - age/cohort/year ranges.';
+
+CREATE TABLE model_version_vaccine (
+  id SERIAL,
+  model_version INTEGER NOT NULL,
+  vaccine TEXT NOT NULL, 
+  activity_type TEXT NOT NULL, 
+  delivery_order INTEGER NOT NULL,
```

Comment:
in the import script you use `order` - please update one or the other.  Does this not risk falling into some tension with the same concept in the responsibilities (via `secenario_coverage_set`?  We don't want the same information duplicated generally.

Reply:
OK. I have removed order/delivery_order. It dose duplicate order in scenario_coverage_set.